setup/db/db/schema-452to460.sql

```diff
@@ -353,6 +353,8 @@ CREATE VIEW `cloud`.`user_vm_view` AS
         `cloud`.`user_vm_details` `custom_speed`  ON (((`custom_speed`.`vm_id` = `cloud`.`vm_instance`.`id`) and (`custom_speed`.`name` = 'CpuSpeed')))
            left join
         `cloud`.`user_vm_details` `custom_ram_size`  ON (((`custom_ram_size`.`vm_id` = `cloud`.`vm_instance`.`id`) and (`custom_ram_size`.`name` = 'memory')));
+        delete s1 from ssh_keypairs s1, ssh_keypairs s2 where s1.id > s2.id and s1.public_key = s2.public_key;
```

Comment:
@sedukull there are no constraints which refer to ssh_keypairs as the foreign key. Yes there are foreign key constraints on the columns of ssh_keypairs, but deleteing them won't create any problem as they are not referenced further.